guides/common/modules/ref_permissions-required-for-provisioning.adoc

```diff
@@ -0,0 +1,43 @@
+:_content-type: REFERENCE
+
+[id="permissions-required-for-provisioning_{context}"]
+= Permissions Required for Provisioning
+
+The following table lists the permissions a non-admin user requires to provision hosts.
+
+.Permissions required for provisioning hosts
+[options="header"]
+|====
+|Resource type|Permissions
+|Compute resource|view_compute_resources, view_compute_resources_vms, create_compute_resources_vms, destroy_compute_resources_vms, power_compute_resources_vms
+|Location|view_locations
+|Subnet|view_subnets
+|Hostgroup|view_hostgroups, create_hostgroups, edit_hostgroups, play_roles_on_hostgroup
+|Organization|view_organizations
+|Compute profile|view_compute_profiles
+|Host|view_hosts, create_hosts, edit_hosts, destroy_hosts, build_hosts, power_hosts, play_roles_on_host, view_discovered_hosts, submit_discovered_hosts, auto_provision_discovered_hosts, provision_discovered_hosts, edit_discovered_hosts, destroy_discovered_hosts
+|Activation Keys|view_activation_keys
+|Domain|view_domains
+//Environment is missing -- at least in the web UI
+|Environment|view_environments
+|Image|view_images
+//I don't see KTEnvironment in the web UI
+|Katello::KTEnvironment|view_lifecycle_environments
+|Medium|view_media
+|Operatingsystem|view_operatingsystems
+|Ptable|view_ptables
+|Provisioning template|view_provisioning_templates
+|Content View|view_content_views
+//I don't see Product in the web UI
+|Katello::Product|view_products
```

Comment:
Only if Katello is used

Reply:
I added this and similar other comments directly to the table with permissions, at least until I find a better way to handle this.